website/content/docs/concepts/oidc-provider.mdx

```diff
@@ -0,0 +1,152 @@
+---
```

Comment:
I think this document works pretty well as a conceptual explainer. However, I wonder if we should write an additional document that walks our users through the story of setting up OIDC provider with code snippets (similar to how our docs usually do this [example](https://www.vaultproject.io/docs/secrets/transform#setup)). 

Do we have plans to write additional documentation around OIDC provider? If not, I think we should adapt the structure of this page a bit and consider moving it out of the "Concepts" section. I would expect the document to live under "Secrets Engines" -> "Identity" -> "OIDC Provider". This matches up with the hierarchy of the API documentation for the feature. We could likely break out the identity token docs into its own section to distinguish the two use cases. Something like "Secrets Engines" -> "Identity" -> "Tokens".

Reply:
@austingebauer , hi. Yes, the education engineer (Lynn) will provide a learn tutorial on how to set up OIDC Provider. I can link to that tutorial from this document.